### PR TITLE
Import tmpop explorer

### PR DESCRIPTION
**Pull request by such**
This change is [Reviewable](https://reviewable.io/reviews/stratumn/indigo-js/189)

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 node_modules
-packages/*/node_modules
\ No newline at end of file
+packages/*/node_modules
+packages/tmpop-explorer
+packages/angular-mapexplorer
+packages/angular2-mapexplorer
```
Comment (such) [inline .dockerignore]: @t-bast this file is a bit out of the scope but you might want to have a look. I think it's useless to include all these project in the Docker build for agent-ui.local and agent-sample.local.
What do you think?

**Comment by t-bast** [comment]
:lgtm:

---

Reviewed 6 of 52 files at r2, 1 of 29 files at r3, 28 of 31 files at r5, 32 of 44 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[.dockerignore, line 5 at r6](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-L0nDw3s3Aw_W4gZwLFv-r6-5:-L0nEDZpeBJkPO116iiI:bbkzjlo) ([raw file](https://github.com/stratumn/indigo-js/blob/1fd637139df297f0a88b3687d70efa45191b1249/.dockerignore#L5)):*
<details><summary><i>Previously, such (Adrien Montfort) wrote…</i></summary><blockquote>

@t-bast this file is a bit out of the scope but you might want to have a look. I think it's useless to include all these project in the Docker build for agent-ui.local and agent-sample.local.
What do you think?
</blockquote></details>

That's a good point! Thanks

---

*[packages/tmpop-explorer/README.md, line 7 at r6](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-L0nElTBPbwKzb-w8CAX:-L0nElTBPbwKzb-w8CAY:b-5oen9g) ([raw file](https://github.com/stratumn/indigo-js/blob/1fd637139df297f0a88b3687d70efa45191b1249/packages/tmpop-explorer/README.md#L7)):*
> ```Markdown
> ## Demo & Examples
> 
> Live demo: [stratumn.github.io/indigoexplorer](http://stratumn.github.io/indigoexplorer/)
> ```

Is that supposed to work?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-:-L0nFdY0MU01B83cl8OK:bnfp4nl)*

**Comment by such** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[packages/tmpop-explorer/README.md, line 7 at r6](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-L0nElTBPbwKzb-w8CAX:-L0nPFG6-EdVa8Ttl9Uv:b-rnka36) ([raw file](https://github.com/stratumn/indigo-js/blob/1fd637139df297f0a88b3687d70efa45191b1249/packages/tmpop-explorer/README.md#L7)):*
<details><summary><i>Previously, t-bast (Bastien Teinturier) wrote…</i></summary><blockquote>

Is that supposed to work?
</blockquote></details>

Good point. I need to try and publish it.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/189)*

**Comment by such** [comment]
Review status: 61 of 65 files reviewed at latest revision, 2 unresolved discussions.

---

*[packages/tmpop-explorer/README.md, line 7 at r6](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-L0nElTBPbwKzb-w8CAX:-L0nRfnkMzbu8v5BUMAk:bv1btlh) ([raw file](https://github.com/stratumn/indigo-js/blob/1fd637139df297f0a88b3687d70efa45191b1249/packages/tmpop-explorer/README.md#L7)):*
<details><summary><i>Previously, such (Adrien Montfort) wrote…</i></summary><blockquote>

Good point. I need to try and publish it.
</blockquote></details>

Well it does not work (because of Lerna I think) and I don't really want to make it work so I removed all references to it!

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/189)*

**Comment by such** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[packages/tmpop-explorer/README.md, line 7 at r6](https://reviewable.io:443/reviews/stratumn/indigo-js/189#-L0nElTBPbwKzb-w8CAX:-L0o25BIeg10svaWViwY:b-896fix) ([raw file](https://github.com/stratumn/indigo-js/blob/1fd637139df297f0a88b3687d70efa45191b1249/packages/tmpop-explorer/README.md#L7)):*
<details><summary><i>Previously, t-bast (Bastien Teinturier) wrote…</i></summary><blockquote>

Sounds good :)
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/189)*

**Pull request merged**